pages/in/widget/pay/index.vue

```diff
@@ -609,37 +609,32 @@ export default {
     async calculateGasFee() {
       let gas;
       let gasPrices;
-      const from = await this.fetchCurrentCosmosWallet();
-      if (!from) return '';
-      const to = this.toUsers[0].cosmosWallet;
       if (this.isMultiSend) {
         const tos = this.toUsers.map(u => u.cosmosWallet);
         const values = [...this.amounts];
         if (this.hasAgentFee) {
           tos.push(this.agentUser.cosmosWallet);
           values.push(this.agentFee);
         }
-        ({ gas, gasPrices } = await cosmosTransferMultiple(
-          {
-            from,
-            tos,
-            values,
-            memo: this.remarks,
-          },
-          null,
-          { simulate: true },
-        ));
+        const fee = {
+          amount: [{
+            denom: COSMOS_DENOM,
+            amount: 1000,
+          }],
+          gas: parseInt(BASIC_GAS, 10),
```

Comment:
This is `isMultiSend` case, should the gas be `HEAVY_GAS` ? Also these gas estimate function should be refactored into cosmos helper